### PR TITLE
Update the plist information for swift toolchains

### PR DESCRIPTION
**Pull request by shahmishal**
rdar://45197621

### CODE REVIEW

**Comment by shahmishal** [comment]
@swift-ci build toolchain macOS

**Comment by shahmishal** [comment]
@swift-ci smoke test

File: utils/build-script-impl [hunk 1]
```diff
@@ -3741,6 +3741,9 @@ function build_and_test_installable_package() {
           call ${PLISTBUDDY_BIN} -c "Add OverrideBuildSettings:ENABLE_BITCODE string 'NO'" "${DARWIN_TOOLCHAIN_INFO_PLIST}"
           call ${PLISTBUDDY_BIN} -c "Add OverrideBuildSettings:SWIFT_DISABLE_REQUIRED_ARCLITE string 'YES'" "${DARWIN_TOOLCHAIN_INFO_PLIST}"
           call ${PLISTBUDDY_BIN} -c "Add OverrideBuildSettings:SWIFT_LINK_OBJC_RUNTIME string 'YES'" "${DARWIN_TOOLCHAIN_INFO_PLIST}"
+          call ${PLISTBUDDY_BIN} -c "Add OverrideBuildSettings:SWIFT_DEVELOPMENT_TOOLCHAIN string 'YES'" "${DARWIN_TOOLCHAIN_INFO_PLIST}"
+          call ${PLISTBUDDY_BIN} -c "Add DefaultBuildSettings dict" "${DARWIN_TOOLCHAIN_INFO_PLIST}"
+          call ${PLISTBUDDY_BIN} -c "Add OverrideBuildSettings:SWIFT_USE_DEVELOPMENT_TOOLCHAIN_RUNTIME string 'YES'" "${DARWIN_TOOLCHAIN_INFO_PLIST}"
```
[the remark this answers is not in the document]
Comment (shahmishal) [reply utils/build-script-impl]: Correct let update this.

**Comment by shahmishal** [comment]
@swift-ci build toolchain macOS

**Comment by shahmishal** [comment]
@swift-ci smoke test

File: utils/build-script-impl (hunk 1 above)
[the remark this answers is not in the document]
Comment (shahmishal) [reply utils/build-script-impl]: @owensd updated

**Review by owensd — APPROVED** [review]
Yeah, that looks good to me.

**Pull request merged**